contracts/libraries/BiMap.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.5.0;
+
+
+library BiMap {
+    struct Data { 
+        mapping(uint16 => address) idToAddress;
+        mapping(address => uint16) addressToId;
+    }
+
+    function hasId(Data storage self, uint16 id) public view returns (bool) {
```

Comment:
What are your thoughts on
1) Naming these function more generic (e.g. `hasLeftKey`, etc) or
2) Naming the BiMap a IdToAddressBiMap

Right now, we have a very generic class with a very ungeneric interface definition.

❓

Reply:
I would choose the second one since, as previously stated, I would not want to discriminate between left and right on a bijective map.